test/session.test.ts

```diff
@@ -1270,4 +1270,100 @@ describe('MediaSession', () => {
             });
         });
     });
+
+    describe('Custom Attributes', () => {
+        it('should propogate Session attributes created Events', () => {
```

Comment:
this sounds a little awkward (and it's propAgate 😀)...maybe the following?
```suggestion
        it('should include session attributes in Events when provided', () => {
```